### ISSUE TITLE
Slow messaging performance after connected a while?

### ISSUE THREAD

**Issue by natevw** [Hexxeh/libpebble#21]
I'm trying to send over an image to the watch as fast as possible. When I first start the repl, I am able to send one in about 2.5 seconds which isn't great but seems reasonable considering it takes over 50 requests to complete (unoptimized data format and lots of other message overhead means I can only send three rows at a time).

However, once the watch has been connected for a while, my app only gets a request through every second or two. So it takes over a minute to fill the screen! Why does the amount of time the watch has been connected affect this?

NOTE: I'm actually using httpebble support via Aleksandyr's branch at https://github.com/aleksandyr/libpebble but given https://github.com/pebble/pebblekit/issues/31 I'm wondering if this is a lower-level Bluetooth issue.

**Comment by Hexxeh** [comment]
It might well not be libpebble specific, possibly just a Bluetooth stack bug on whatever device you're using?

**Comment by rjcase** [comment]
@natevw yes, it's likely that it's caused by BT sniff (power-saving) mode described in https://github.com/pebble/pebblekit/issues/31
